Xamarin.Forms.Platform.iOS/Renderers/ShellFlyoutContentManager.cs

```diff
@@ -0,0 +1,261 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using CoreAnimation;
+using CoreGraphics;
+using Foundation;
+using UIKit;
+
+namespace Xamarin.Forms.Platform.iOS
+{
+	class ShellFlyoutContentManager
+	{
+		double _headerMin = 56;
+		double _headerOffset = 0;
+		UIView _contentView;
+		UIScrollView ScrollView => _contentView as UIScrollView;
+		UIContainerView _headerView;
+		UIView _footerView;
+		double _headerSize;
+		readonly IShellContext _context;
+
+		IShellController ShellController => _context.Shell;
+		public ShellFlyoutContentManager(IShellContext context)
+		{
+			_context = context;
+			_context.Shell.PropertyChanged += OnShellPropertyChanged;
+			ShellController.StructureChanged += OnStructureChanged;
+		}
+
+		public View Content
+		{
+			get;
+			set;
+		}
+
+		public UIView ContentView
+		{
+			get
+			{
+				return _contentView;
+			}
+			set
+			{
+				if (ScrollView != null && !(ScrollView is UITableView))
+					ScrollView.Scrolled -= ScrollViewScrolled;
+
+				_contentView = value;
+
+				if (ScrollView != null && !(ScrollView is UITableView))
+					ScrollView.Scrolled += ScrollViewScrolled;
+
+				if (ScrollView != null && Forms.IsiOS11OrNewer)
+					ScrollView.ContentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentBehavior.Never;
```

Comment:
```suggestion
                                if (ScrollView == null)
                                      return;
				
				if (!(ScrollView is UITableView))
					ScrollView.Scrolled += ScrollViewScrolled;

				if (Forms.IsiOS11OrNewer)
					ScrollView.ContentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentBehavior.Never;
```

Reply:
The content might not be a ScrollView type